### PR TITLE
feat: ui update for new flow

### PR DESCRIPTION
**Pull request by alvindaiyan**
## Description

ui change for new data flow:
- removed all select models field
- add magic button for generate

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

File: scripts/main.py [hunk 1]
```diff
@@ -297,6 +294,41 @@ def encode_no_json(obj):
             response = requests.put(f'{url}inference/v2/{inference_id}/run', json=payload, headers={'x-api-key': api_key})
             response.raise_for_status()
             self.current_inference_id = inference_id
+        elif upload_param_response['status'] != 200:
+            err = upload_param_response['error']
```
Comment (yike5460) [inline scripts/main.py:298]: what is the the !200 response? is this to judge if selected model was existed in s3 bucket?

**Review by yike5460 — COMMENTED** [review]
we should add logic to check if model was existed on s3 bucket in hijacked generate function before calling the inference api

**Comment by alvindaiyan** [comment]
> we should add logic to check if model was existed on s3 bucket in hijacked generate function before calling the inference api

this logic is within the first api call:

```
            response = requests.post(f'{url}inference/v2', json=payload, headers={'x-api-key': api_key})

```

File: scripts/main.py (hunk 1 above)
Comment (alvindaiyan) [reply scripts/main.py]: if !200, it means there is some errors.

**Review by AoyuQC — APPROVED** [review]

**Pull request merged**